tt_metal/core_descriptors/grayskull_120_arch.yaml

```diff
@@ -31,7 +31,7 @@ E150:
       [[2, -1],[3, -1],[4, -1],[5, -1],[8, -1],[9, -1],[10, -1],[11, -1]]
 
     dispatch_cores:
-      [[0, -1], [1, -1], [6, -1], [7, -1]]
+      [[0, -1], [6, -1], [1, -1], [7, -1]]
```

Comment:
Riskiest change in entire pr. For 2 cqs, the placement of prefetcher and dispatcher is now changed, such that cq0 has same placement as when only 1 cq is initialized.